Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,31 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testDecodePlist() {
+        let point = CGPoint()
+        let temporaryDirectoryURL = FileManager.default.temporaryDirectory
+        let temporaryFilename = ProcessInfo().globallyUniqueString
+        let fileURL = temporaryDirectoryURL.appendingPathComponent(temporaryFilename)
+        do {
+            let encoder = PropertyListEncoder()
+            let dataToEncode = try encoder.encode(point)
+            FileManager.default.createFile(atPath: fileURL.path, contents: dataToEncode, attributes: nil)
+            
+            XCTAssert(FileManager.default.fileExists(atPath: fileURL.path))
+            
+            let fileContents = FileManager.default.contents(atPath: fileURL.path)
```

Comment:
You're actually meant to be using the `decodePlist(as:using:)` function you created in this test 🤣